src/tests/system/tests/test_ipa.py

```diff
@@ -0,0 +1,87 @@
+"""
+SSSD IPA tests.
+
+:requirement: ipa
+"""
+
+from __future__ import annotations
+
+import time
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.ipa import IPA
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.ticket(bz=1926622)
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa__add_test_to_verify_authentication_indicators(client: Client, ipa: IPA):
+    """
+    :title: Add support to verify authentication indicators
+    :description:
+        Kerberos client may have different means to prove possession of a client principal
+        credentials to a KDC. There are several so-called ‘pre-authentication mechanisms’ that are
+        used for this purpose. FreeIPA KDC is able to record which pre-authentication method was used
+        when issuing the ticket granting ticket. The specific label is called an authentication indicator.
+    :setup:
+        1. Add pam_sss_gss configuration to /etc/sssd/sssd.conf
+        2. Add pam_sss_gss.so to /etc/pam.d/sudo
+        3. Restart SSSD
+        4. Enable SSSD debug logs
+    :steps:
+        1. Switch to ipa user
+        2. obtain Kerberos ticket.
+        3. Create sudo configuration that allows an admin to
+           run SUDO rules
+        4. Try 'sudo -l' as admin
+        5. As root, check content of sssd_pam.log
+       `6. Check if acquired service ticket has req. indicators: 0
```

Comment:
The steps don't match the actual code, if it did, it'd look like this. 

```
5. Add pam_sss_gss configuration to /etc/sssd/sssd.conf
6. Try 'sudo -l' as admin
7. Check if acquired service ticket has req. indicators: 0
8. Check if acquired service ticket has req. indicators: 2
```

I prefer what you have in the steps, you can reuse the variable name, if you want to do it this way that's fine, just update the steps to match the code. 

```
    log = client.fs.read(client.sssd.logs.pam)
    with client.ssh(user.name, password) as ssh:
        ssh.run(f"kinit {user.name}@{ipa.host.realm}", input=password)
        ssh.run("klist")
        ssh.disconnect()
    assert not client.auth.sudo.list(user.name)
    assert "indicators: 0" in log 
    
    # change config

redo the same steps, but change the last assertion. 
```